File: zalohovaci_system_api/Models/BackupJob.cs
namespace zalohovaci_system_api.Models
{
    public class BackupJob
    {
        public int Id { get; set; }
        public string jobId { get; set; }
        public int retention { get; set; }
        public int method { get; set; }
        public string timing { get; set; }
        public DateTime createdAt { get; set; }
    }
}

File: zalohovaci_system_admin/src/app/models/backup-job.ts

export class BackupJob {
  public id: number;
  public jobId: string;
  public sources: string[];
  public targets: string[];
  public retentionSize: number;
  public retentionCount: number;
  public method: string;
  public timing: string;
  public createdAt: string;
}
